src/Magento/FunctionalTestingFramework/DataGenerator/Handlers/DataObjectHandler.php

```diff
@@ -297,4 +338,32 @@ private function extendDataObject($dataObject)
         }
         return $dataObject;
     }
+
+    /**
+     * Output to the log a summary of data entity name and data entity key violations.
+     *
+     * @throws TestFrameworkException
+     * @return void
+     */
+    private function logViolationSummary()
+    {
+        $nameViolationCount = count($this->nameViolations);
+        if ($nameViolationCount > 0) {
+            LoggingUtil::getInstance()->getLogger(self::class)->notification(
+                "{$nameViolationCount} data entity name violations detected. Data entity names must be"
```

Comment:
To direct users to detailed log, how how about appending "See mftf.log for details." to notice messages

Reply:
fixed